### PR TITLE
gittuf OpenSSF Sandbox Application

### PR DESCRIPTION
**Pull request by adityasaky**
gittuf is seeking admission to the OpenSSF Sandbox. We aim to be part of the Supply Chain Integrity WG, where we presented gittuf in the August 30th, 2023 meeting.

## Overview

gittuf implements security controls into Git repositories using existing Git semantics like its support for cryptographic signatures and its content addressed store. By embedding source security policies into the repository, gittuf makes policies transparent that enables _distributed verification_ by all repository users. Further, as gittuf versions and tracks changes to policies using Git semantics, past repository states can be _audited_ against the then-applicable policies.

## Sandbox Requirements

We believe gittuf meets the Sandbox entry requirements.

### Maintainers

gittuf currently has four maintainers across three organizations:
* Aditya Sirish A Yelgundhalli (@adityasaky, New York University)
* Billy Lynch (@wlynch, Chainguard)
* Justin Cappos (@JustinCappos, New York University)
* Reza Curtmola (@reza-curtmola, New Jersey Institute of Technology)

### Alignment to the OpenSSF

The mission of the gittuf project aligns with the [OpenSSF's Technical Vision](https://openssf.org/about/), especially with:

> Developers, auditors, and regulators can create and easily distribute security policies that are enforced through tooling and automation, providing continuous assurance of the results.

gittuf also has synergy with the mission of the [OpenSSF Supply Chain Integrity Working Group](https://github.com/ossf/wg-supply-chain-integrity):

> scalable standardized attestable practices for supply chain security

gittuf provides a framework to implement such practices for securing Git repositories, and is complementary to other efforts under the working group like SLSA.

### License Review

All gittuf repositories (implementation, website, and demo) use the Apache-2.0 license. The review found no conflicts.

Issue: https://github.com/ossf/tac/issues/199

## Links

Repository: https://github.com/gittuf/gittuf
Website: https://gittuf.github.io
Demo: https://github.com/gittuf/demo

### CODE REVIEW

File: process/project-lifecycle-documents/gittuf_sandbox_stage.md [hunk 1]
```diff
@@ -76,8 +76,8 @@ badge.
 
 ### IP policy and licensing due diligence
 
-License due diligence is applicable. An issue will be opened alongside this
-application to track that process.
+License due diligence is applicable. We have requested the IP policy and license
+review from the OpenSSF [here](https://github.com/ossf/tac/issues/199).
```
Comment (jsoref) [inline process/project-lifecycle-documents/gittuf_sandbox_stage.md:80]: I'm not a member of the org, but from an [a11y](https://www.a11yproject.com/) perspective, [`here` links are considered harmful](https://granicus.com/blog/why-click-here-links-are-bad/#:~:text=Click%20Here%20Links%20are%20Bad%20for%20Accessibility&text=If%20all%20of%20your%20links,to%20establish%20context%20for%20links.)

Comment (adityasaky) [reply process/project-lifecycle-documents/gittuf_sandbox_stage.md]: Thanks, addressed! [45ded85](https://github.com/ossf/tac/pull/198/commits/45ded85f3c3b7784b9ad6882feef35fc5ad0fbd3)

File: process/project-lifecycle-documents/gittuf_sandbox_stage.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+| Website                                | https://gittuf.github.io                                                                             |
+| Contributing guide                     | https://github.com/gittuf/gittuf/blob/main/CONTRIBUTING.md                                           |
+| Roadmap                                | https://github.com/gittuf/gittuf/blob/main/docs/roadmap.md                                           |
+| Demos                                  | https://github.com/gittuf/demo                                                                       |
+| Presentation to the SCI WG             | https://docs.google.com/presentation/d/12ivx9LwMe1xgOvazMqXeJpkmK80Z-rhD1_CIy8T2d6I/edit?usp=sharing |
+| Video Recording of SCI WG Presentation | https://zoom.us/rec/play/X1-avN0pWIzxWRQLpFSlDcjWVVsrZvdXta_6lK88Gvz2DbBReDYVJgDaDigdHAb0eoBvzhhBYQEaDWpt.CkXNT4amJEseG6q8?canPlayFromShare=true&from=share_recording_detail&continueMode=true&componentName=rec-play&originRequestUrl=https%3A%2F%2Fzoom.us%2Frec%2Fshare%2FvAJaAIQ7KvVjgnk5ykNZCooTAl4L07e8escBmgqfHmxY4d6FJEkPkmwyPnSHRg8N.ZCfEGWv0FMBG-a9K |
```
Comment (lehors) [inline process/project-lifecycle-documents/gittuf_sandbox_stage.md:97]:
```suggestion
| Video Recording of SCI WG Presentation | https://zoom.us/rec/play/X1-avN0pWIzxWRQLpFSlDcjWVVsrZvdXta_6lK88Gvz2DbBReDYVJgDaDigdHAb0eoBvzhhBYQEaDWpt.CkXNT4amJEseG6q8 |
```

**Review by steiza — APPROVED** [review]

**Comment by steiza** [comment]
Sorry! I mistakenly though I was just reviewing https://github.com/ossf/tac/pull/198/commits/314df730d3301c765e7227a4b48155d538774f7d

**Review by steiza — APPROVED** [review]
I think this is a promising project to accept at the Sandbox stage!

**Review by lehors — APPROVED** [review]

**Pull request merged**